setup/pi/create-backingfiles.sh

```diff
@@ -99,9 +99,9 @@ function add_drive () {
   log_progress "Creating filesystem with label '$label'"
   if [ "$useexfat" = true  ]
   then
-    mkfs.exfat "$loopdev" -L "$label"
+    mkfs.exfat "$loopdev" -n "$label"
```

Comment:
Correct, on armbian it would fail and say it needed the -I to proceed.
mkfs.vfat 4.2 (2021-01-31)
Exfat was not installed prior and i was using vfat because i knew it was in my image before starting teslausb.

Reply:
which version of Armbian is that on then? I don't have an Orange Pi, but installed a recent Armbian on Rock Pi 4C+, and it uses mkfs.exfat 1.1.3, which does not have that `-n` option:
```
root@teslausb:~# uname -a
Linux teslausb 5.15.93-rockchip64 #23.02.2 SMP PREEMPT Fri Feb 17 23:48:36 UTC 2023 aarch64 aarch64 aarch64 GNU/Linux
root@teslausb:~# cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.1 LTS"
root@teslausb:~# mkfs.exfat --help
Usage: mkfs.exfat
        -L | --volume-label=label                              Set volume label
        -c | --cluster-size=size(or suffixed by 'K' or 'M')    Specify cluster size
        -b | --boundary-align=size(or suffixed by 'K' or 'M')  Specify boundary alignment
             --pack-bitmap                                     Move bitmap into FAT segment
        -f | --full-format                                     Full format
        -V | --version                                         Show version
        -v | --verbose                                         Print debug
        -h | --help                                            Show help
root@teslausb:~# mkfs.exfat -V
exfatprogs version : 1.1.3
```

Reply:
23.05.0-trunk Bullseye 4.9.318-sun50iw9
The 4.9 kernel here, as the “main line” kernel doesnt have wifi support.
https://www.armbian.com/orange-pi-zero-2/

Reply:
Does the 4.9 kernel actually work for you for TeslaUSB? I seem to remember the xfs reflink support that TeslaUSB needs wasn't added until later.

Reply:
I think youre right as its not uploaded videos in the past few days and clips inside the car only shows the current drive no previous videos so need to try something else.

Reply:
on this new image orange pi debian 10.2 kernel 4.9
exfatprogs not available
exfat-utils gives me mkexfatfs 1.3.0 and for mkexfatfs the option for label is n not L, L not even available for option.